src/convertToCase/getErrors.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const getErrors = (currText, toCase) => {
+  const payload = [];
+  const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  if (!currText) {
+    payload.push({
+      message: 'Text to convert is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!toCase) {
+    payload.push({
+      message: '"toCase" query param is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (toCase && !availableCases.includes(toCase)) {
+    payload.push({
+      message: 'This case is not supported. '
```

Comment:
would be nice to create an object with possible errors

Reply:
Did it)